easybuild/easyconfigs/p/PyCogent/PyCogent-1.9-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'PythonPackage'
+
+name = 'PyCogent'
+version = '1.9'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://%(namelower)s.org'
+description = """PyCogent is a software library for genomic biology. It is a fully
+ integrated and thoroughly tested framework for: controlling third-party applications;
+ devising workflows; querying databases; conducting novel probabilistic analyses of
+ biological sequence evolution; and generating publication quality graphics."""
```

Comment:
This is different from the existing `PyCogent` easyconfigs, but I like this a lot better.

Do you mind changing it to this in the existing `PyCogent` easyconfigs (makes sense to do that in this PR imho)?

Reply:
@boegel I added this to the existing easyconfigs.